static/js/app.js

```diff
@@ -114,11 +136,12 @@ textArea.on('mouseup', function (e) {
 textArea.on('contextmenu', function (e) {
   event.preventDefault();
   let position = textArea[0].selectionStart;
-  let annotations = tagModel.getAnnotationsAtPos(position);
-  if (annotations.length > 0) {
+  deleteList = tagModel.currentDoc.getAnnotationsAtPos(position);
+
+  if (deleteList.length > 0) {
     $('#delete-menu').append('<h6>Delete Annotation:</h6><hr style="margin: 0;">')
```

Comment:
#delete-menu is duplicated a lot, you should store it in a variable

Reply:
updated